### PR TITLE
Check delay number

### PR DESCRIPTION
**Pull request by bbuzz31**
Addresses #481

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
++ Ensure both delays are written for the two SAR dates in the GUNW
```
Comment (dbekaert) [inline CHANGELOG.md:9]: Correct this:
- Are we writing SAR delays int he GUNW? We are writing the interferometric hydrostatic and wet delays. 
- Also it more like at asset statement that captures failing XXX

File: tools/RAiDER/cli/raider.py [hunk 2]
```diff
@@ -435,6 +435,8 @@ def calcDelaysGUNW():
     # return a list with the path to cube for each date
     cube_filenames = calcDelays([path_cfg])
 
+    assert len(cube_filenames) == 2, 'Incorrect number of delay files written.'
```
[the remark this answers is not in the document]
Comment (bbuzz31) [reply tools/RAiDER/cli/raider.py:438]: Which delay failed is buried further down in RAiDER, and would be non-trivial to capture.
The date that fails is caught and output to the error log.

These are absolute delays, an intermediate step before the interferometric is calculated in the calcGUNW script.

File: CHANGELOG.md (hunk 1 above)
Comment (bbuzz31) [reply CHANGELOG.md:9]: We are writing one netcdf file for each SAR date in the GUNW for subsequent processing. I''ll update to reflect that it's just an assert statement

Comment (dbekaert) [reply CHANGELOG.md]: You mean you are having two intermediate netcdf files, these are then used to compute interferometric, and only interferometric is added to gunw?

Comment (bbuzz31) [reply CHANGELOG.md]: Yup

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-+ Ensure both delays are written for the two SAR dates in the GUNW
++ Add assert statement to raise error if the delay cube for each SAR date in a GUNW IFG is not written
```
[the remark this answers is not in the document]
Comment (bbuzz31) [reply CHANGELOG.md:9]: No that's correct. the assert statement is checking that each of the two intermediate files is written

**Review by dbekaert — APPROVED** [review]
LGTM

**Pull request merged**